addons/mail/static/src/components/thread_view/thread_view.xml

```diff
@@ -10,6 +10,15 @@
                     </div>
                 </t>
                 <t t-elif="threadView.threadCache.isLoaded or threadView.thread.isTemporary">
+                    <t t-if="threadView.nonEmptyMessages.length === 0 and (threadView.thread.activities.length or threadView.threadViewer and threadView.threadViewer.chatter and threadView.threadViewer.chatter.isAttachmentBoxVisible)">
+                        <div class="o_MessageList_empty_separator">
+                            <hr class="o_MessageList_empty_separatorLine"/>
+                            <span class="o_MessageList_empty_separatorText">
+                                Messages
+                            </span>
+                            <hr class="o_MessageList_empty_separatorLine"/>
+                        </div>
+                    </t>
```

Comment:
Attachment box is concern of chatter, and chatter is one kind of container of thread view. This means the `ThreadView` should not have chatter concern (including the attachment box).

The separator should be defined in `Chatter` component template.